pelican/plugins/seo/seo_enhancer/html_enhancer/__init__.py

```diff
@@ -88,7 +88,25 @@ def __init__(self, file, output_path, path, open_graph=False, twitter_cards=Fals
                 locale=_settings.get("LOCALE"),
             )
 
+            if isinstance(file, Article):
+                _modified = getattr(file, "modified", None)
+                _author_profiles = _settings.get(
+                    "SEO_ENHANCER_AUTHOR_FACEBOOK_PROFILES", {}
+                )
+                _author_profile = _author_profiles.get(_author.name, None)
+                self.open_graph_article = OpenGraphArticle(
+                    date=_date,
+                    modified=_modified,
+                    category=_category.name,
+                    tags=getattr(file, "tags", []),
+                    author=_metadata.get("fb_profile") or _author_profile,
```

Comment:
Same here, I think we should have two tests (or a parametrized one) [here](https://github.com/pelican-plugins/seo/blob/main/pelican/plugins/seo/tests/test_seo_enhancer.py) to ensure that the open graph element is filled by both `SEO_ENHANCER_AUTHOR_FACEBOOK_PROFILES` and `SEO_ENHANCER_AUTHOR_TWITTER_PROFILES` new settings if filled, or is filled by using fallback to `_author_profile`.